packages/bee-q/src/components/breadcrumb-item/scss/bq-breadcrumb-item.scss

```diff
@@ -0,0 +1,19 @@
+/* -------------------------------------------------------------------------- */
+/*                                Breadcrumb item styles                */
+/* -------------------------------------------------------------------------- */
+
+@import './bq-breadcrumb-item.variables';
+
+.breadcrumb-item {
+  @apply font-regular leading-regular text-text-secondary no-underline focus-visible:focus active:outline-none;
+  @apply px-[var(--bq-breadcrumb-item--paddingX)] py-[var(--bq-breadcrumb-item--paddingY)] text-[length:var(--bq-breadcrumb-item--font-size)];
```

Comment:
We should use CSS logical properties more instead of padding left/right.
https://tailwindcss.com/docs/padding#using-logical-properties